src/reducers/commonInitialState.js

```diff
@@ -0,0 +1,5 @@
+import { fromJS } from 'immutable';
+
+export default fromJS({
```

Comment:
It's a matter of taste but I prefer to have the initial state inside the commonReducer, thus it's easy to check.